css/main.css

```diff
@@ -754,7 +757,69 @@ h2 {
 }
 
 #footer a:hover {
-  color:#1B7B98
+  color:#05cc47
+}
+
+
+/* Added By Aakash
+The Flash Page CSS*/
+
+body, html {
+  height: 100%;
+}
+
+.courosal {
+  animation-name: yo;
+  animation-duration: 10s;
+  animation-iteration-count: infinite;
+  transition: all .5s;
+  height: 100%;
+  width: 100%;
+  background-size: cover;
+  z-index: +1000;
```

Comment:
Not sure would this work as intended or not.
`z-index` only works with positioned elements .
Best reference for z-index https://tympanus.net/codrops/css_reference/z-index/
cc @Manan-YMCA/elementsculmyca2017-festteam